app/models/managed_interactive.rb

```diff
@@ -45,6 +45,7 @@ class ManagedInteractive < ActiveRecord::Base
   has_one :labbook, :as => :interactive, :class_name => 'Embeddable::Labbook'
 
   belongs_to :linked_interactive, :polymorphic => true
+  belongs_to :converted_interactive, :polymorphic => true, :foreign_key => :legacy_ref_id, :foreign_type => :legacy_ref_type
```

Comment:
I'd name this `legacy_embeddable`. It is the converted_interactive that has a legacy embeddable. 

You could possibly add the inversion of this of on each of the legacy embeddables like
`has_one :converted_interactive, as: :legacy_embeddable`

With that inverted relationship, it might be possible to go back to iterating over activities and use an `includes` that would load the converted_interactive of every embeddable in the activity. But I'm not entirely sure it would work.  Without going back to that I don't know an efficient way tell when all of the items of an activity have been re-wired. My best guess would be to use some grouping by activity, but I don't really know how to do that.